opening_hours.js

```diff
@@ -6728,7 +6728,7 @@
         function getWeekNumber(d) {
             // Copy date so don't modify original
             d = new Date(+d);
-            d.setHours(0,0,0);
+            d.setHours(0,0,0,0);
```

Comment:
When I revert this change, the added unit tests still pass using NodeJS v0.10.29. Is that just because of my legacy Node version? (hm, also tested with v4.6.1 with the same result). Can you update the tests to catch that or am I missing something?